src/SonarScanner.MSBuild.PreProcessor/ProcessedArgs.cs

```diff
@@ -62,6 +62,16 @@ public class ProcessedArgs
         /// </summary>
         public bool InstallLoaderTargets { get; private set; }
 
+        /// <summary>
+        /// Path to the Java executable in the archive.
```

Comment:
Could be extracted from the archive or user-provided. I would simply say:
```suggestion
        /// Path to the Java executable.
```